### PR TITLE
Add translation for receive_backup during multi device setup

### PR DESCRIPTION
**Pull request by ralphtheninja**
Fixes

image_2023-08-09_14-59-17

### CODE REVIEW

File: _locales/_untranslated_en.json [hunk 1]
```diff
@@ -56,6 +56,9 @@
   "show_webxdc_in_chat": {
     "message": "Show Private App in Chat"
   },
+  "receive_backup": {
+    "message": "Receiving backup from second device"
```
Comment (ralphtheninja) [inline _locales/_untranslated_en.json:60]: Maybe some other wording is better? Idk.

Comment (farooqkz) [reply _locales/_untranslated_en.json:60]: Dunno about other wording but "a second device" is correct.

**Review by r10s — CHANGES_REQUESTED** [review]
no need to add a new string, there is already a string, but that is just not used on desktop.

so instead of adding a new string, use `multidevice_receiver_titlle` instead of `receive_backup` as described in https://github.com/deltachat/deltachat-desktop/issues/3178

things are in sync with android/ios then as well.

maybe also target the other open points of https://github.com/deltachat/deltachat-desktop/issues/3178 when on that?

**Comment by Simon-Laux** [comment]
> maybe also target the other open points of https://github.com/deltachat/deltachat-desktop/issues/3178 when on that?

that is a big bigger change as it needs a new qr code dialog.

**Comment by ralphtheninja** [comment]
> no need to add a new string, there is already a string, but that is just not used on desktop.
> 
> so instead of adding a new string, use `multidevice_receiver_titlle` instead of `receive_backup` as described in #3178
> 
> things are in sync with android/ios then as well.
> 
> maybe also target the other open points of #3178 when on that?

I can tweak this PR to do that instead.

**Comment by r10s** [comment]
> that is a big bigger change as it needs a new qr code dialog.

i do not really buy that changing a string requires a larger refactoring 🧐 :)

did a pr that fixes the long standing issue and adds missing guidance so that ppl starting "Add Second Device" from the welcome screen have a  chance to succeed: https://github.com/deltachat/deltachat-desktop/pull/3350

**Comment by Simon-Laux** [comment]
> @Simon-Laux however, see that as an experiment from my side, feel free to close or do sth else as you like 😀

I might be too much of an perfectionist, but I can see that this basic subtitle already improves things a lot, so thanks for the pr.

**Pull request merged**